src/main/webapp/i18n/de/submission.json

```diff
@@ -21,7 +21,8 @@
             "result": "Ergebnis",
             "participation": "Participation",
             "lockedSubmissionsLimitReached": "Du hast die maximale Anzahl von unvollständigen Bewertungen in diesem Kurs erreicht. Bitte beende deine offenen Bewertungen, bevor du eine neue beginnen kannst.",
-            "resultTimeout": "Das Ergebnis zu deiner letzten Abgabe ist nicht verfügbar. Bitte gebe deinen Code erneut ab."
+            "resultTimeout": "Das Ergebnis zu deiner letzten Abgabe ist nicht verfügbar. Bitte gebe deinen Code erneut ab.",
+            "createExternal": "Externe Submission erstellen"
```

Comment:
We should be consistent: "Externe Submission" or "Externe Einreichung"

Reply:
I think the consensus was to go with `Einreichung` for submission in German, right? I updated all entries in that file that had submission in it or were otherwise completely untranslated.